contracts/oracle/DynamicVoltRateModel.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.13;
+
+/// @notice The Dynamic VOLT Rate Model exposes a pure function that
+/// allows the DynamicVoltSystemOracle to set a dynamic VOLT rate based
+/// on the current rate and percentage of liquid reserves. Lower liquid
+/// reserves should increase the VOLT rate to incentivize new deposits
+/// and improve the liquidity available for redemptions.
+/// This model can we swapped out by governance, by deploying and new
+/// model and pointing the DynamicVoltSystemOracle to the new model.
+/// @author Eswak, Elliot Friedman
+contract DynamicVoltRateModel {
+    /// @notice the percentage at which the Volt rate will start going above
+    /// the base rate. At less than 30% liquid reserves, rate jumps.
+    uint256 public constant LIQUIDITY_JUMP_TARGET = 0.3e18; // 30%
```

Comment:
this should be an immutable set during construction so we don't have to change code to change the rate